SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialDebuggerEditor.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Utils/SpatialDebugger.h"
```

Comment:
we prefer to split gdk includes to their own group before engine includes
```suggestion
#include "Utils/SpatialDebugger.h"

#include "CoreMinimal.h"
```